<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: ClockstaR: A package for molecular clock partition selection</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link rel="stylesheet" type="text/css" href="R.css">
</head><body>
<h1> ClockstaR: A package for molecular clock partition selection
<img class="toplogo" src="../../../doc/html/logo.jpg" alt="[R logo]">
</h1>
<hr>
<div align="center">
<a href="../../../doc/html/packages.html"><img src="../../../doc/html/left.jpg" alt="[Up]" width="30" height="30" border="0"></a>
<a href="../../../doc/html/index.html"><img src="../../../doc/html/up.jpg" alt="[Top]" width="30" height="30" border="0"></a>
</div><h2>Documentation for package &lsquo;clockstar&rsquo; version 0.1</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
</ul>

<h2>Help Pages</h2>


<table width="100%">
<tr><td width="25%"><a href="clockstar-package.html">clockstar-package</a></td>
<td>ClockstaR: Selecting partitioning strategies for relaxed phylogenetic analysis</td></tr>
<tr><td width="25%"><a href="clockstar-package.html">clockstar</a></td>
<td>ClockstaR: Selecting partitioning strategies for relaxed phylogenetic analysis</td></tr>
<tr><td width="25%"><a href="convert.to.fasta.html">convert.to.fasta</a></td>
<td>This function is used to convert nexus files to fasta format for clockstar execution</td></tr>
<tr><td width="25%"><a href="cut.trees.beta.html">cut.trees.beta</a></td>
<td>cut.trees.beta calculated the diameter of a dendrogram. If the diameter of the dendrogram is higher than that specified by the user, the dendrogram is cut at its longest edge.</td></tr>
<tr><td width="25%"><a href="diagnostics.clockstar.html">diagnostics.clockstar</a></td>
<td>diagnostics.clockstar</td></tr>
<tr><td width="25%"><a href="get.all.groups.html">get.all.groups</a></td>
<td>This function finds the best number of partitions (k) for a data set, and the data subsets in each partitions, as described in Duchene et al.</td></tr>
<tr><td width="25%"><a href="get.all.groups.k.html">get.all.groups.k</a></td>
<td>This function finds the best partitioning strategy for a fixed number of partitions, as described in Duchene et al.</td></tr>
<tr><td width="25%"><a href="min.dist.topo.html">min.dist.topo</a></td>
<td>This function estimates the sBSDmin value described in Duchene et al.</td></tr>
<tr><td width="25%"><a href="min.dist.topo.mat.html">min.dist.topo.mat</a></td>
<td>Estimate the pairwise BSDmin for a list of trees</td></tr>
<tr><td width="25%"><a href="min.dist.topo.mat.para.html">min.dist.topo.mat.para</a></td>
<td>Estimate the pairwise BSDmin for a list of trees</td></tr>
<tr><td width="25%"><a href="optim.edge.lengths.html">optim.edge.lengths</a></td>
<td>This function estimates the branch lengths of a fixed tree topology for multiple alignment data sets stored in a folder.</td></tr>
<tr><td width="25%"><a href="run.clockstar.html">run.clockstar</a></td>
<td>This function is wrapper for the individual ClockstaR functions. It uses a folder that contains the individual gene alignments and the phylogenetic tree to calculate the BSDmin distance.</td></tr>
</table>
</body></html>
